.github/workflows/build-and-push-dev-images.yml

```diff
@@ -27,13 +27,13 @@ on:
         required: false
       lang:
         description: List of languages to build
-        default: 'golang, nodejs, php, python, ruby'
+        default: 'compat, golang, nodejs, php, python, ruby'
```

Comment:
This introduce dependencies when images start using compat.
A change in compat should trigger a rebuild for for the compat based, similarly a change in _upstream_ will require a compat rebuild beforehand.

I would suggest to get this PR through now and tackle the potential rebuild issues in a separate PR...

Reply:
That makes sense.